inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -80,8 +89,19 @@ void FrontEnd::parseInterp(const Model& model, const ie::CNNLayerPtr& layer, con
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "Interp stage with name {} must have only 1 output, "
                      "actually provided {}", layer->name, outputs.size());
+    ie::details::CaselessEq<std::string> cmp;
+    const auto coord = layer->GetParamAsString(g_coordinate_transformation_mode, g_half_pixel);
+    const auto interpMode = layer->GetParamAsString(g_mode, g_linear);
+    InterpolateCoordTransMode coordinateTransMode = InterpolateCoordTransMode::HalfPixel;
+    InterpolateMode mode = InterpolateMode::Linear;
 
-    _stageBuilder->addInterpStage(model, layer->name, layer, layer->GetParamAsInt("align_corners", 0), inputs[0], outputs[0]);
+    coordinateTransMode = coordTransformModeMap.at(coord);
+
+    if (cmp(interpMode, g_linear_onnx)) {
```

Comment:
Can we also create a map for `InterpolateMode`? 
Is it true that InterpolateMode can be only `Linear` or `LinearOnnx` here? 
If yes, you can change this condition to 
```
const auto interpModeString = layer->GetParamAsString(g_mode, g_linear);
const auto interpModeIt = interpModeMap.find(interpModeString);
VPU_THROW_UNLESS(interpModeIt != interpModeMap.end(), ...);
VPU_THROW_UNLESS(*interpModeIt == InterpolateMode::Linear || *interpModeIt  == InterpolateMode::LinearOnnx, ...);

_stageBuilder->addInterpStage(model, layer->name, layer, layer->GetParamAsInt(g_align_corners, 0), *interpModeIt, coordinateTransMode, inputs[0], outputs[0]);
```

Reply:
sure, will do. Thanks

Reply:
done